src/modules/analysis/AnalysisViewComponent.js

```diff
@@ -58,6 +54,18 @@ const getTableRow = a => {
   );
 };
 
+const FieldGroup = ({ id, label, md, value }) => (
```

Comment:
It's not a good idea to have naming conflicts with existing component in react-bootstrap. This will just lead to confusion in the future.

https://react-bootstrap.github.io/components.html#forms-controls